src/bun.js/node/node_fs.zig

```diff
@@ -4924,14 +4924,12 @@ pub const NodeFS = struct {
 
         var times = [2]std.c.timeval{
             .{
-                .tv_sec = args.mtime,
-                // TODO: is this correct?
-                .tv_usec = 0,
+                .tv_sec = args.mtime.seconds,
+                .tv_usec = args.mtime.nanos * 1000,
```

Comment:
This could overflow very easily. Maybe change `nanos` to an `isize` and truncate?

Reply:
@dylan-conway 
Thanks for the advice. You are exactly right. I am now re-implementing it in a way that uses i64, but it is taking a bit of time to fix.